api/src/main/java/io/opentelemetry/trace/BlankSpan.java

```diff
@@ -31,13 +35,18 @@
 @Immutable
 public final class BlankSpan implements Span {
   /**
-   * Singleton instance of this class.
+   * An instance of this class. If there is no {@code SpanContext} or {@link SpanContext#BLANK} to
+   * propagate this instance is used.
    *
    * @since 0.1.0
    */
-  public static final Span INSTANCE = new BlankSpan();
+  public static final Span INSTANCE = new BlankSpan(SpanContext.BLANK);
+
+  private final SpanContext spanContext;
 
-  private BlankSpan() {}
+  BlankSpan(SpanContext spanContext) {
```

Comment:
@bogdandrutu you had a comment on this in my previous PR https://github.com/open-telemetry/opentelemetry-java/pull/258#discussion_r282696358.

This class is exclusively used by `NoopTrace` https://pastebin.com/wTzBX4ve, therefore it makes sense to make it hold the context when necessary. In the cases when there is no context we can use the same instance.

Reply:
I am good, because we don't expose this.